src/sentry/api/serializers/snuba.py

```diff
@@ -314,5 +314,7 @@ def serialize(self, result, column="count"):
 
         if result.data.get("totals"):
             res["totals"] = {"count": result.data["totals"][column]}
+        if "order" in result.data:
```

Comment:
can't use `result.data.get("order")` since order starts at `0` and is falsey

Reply:
Hmm, yeah I could add that too. in a multiaxis topevent query would we want the result to be like this?:
```
{
    "result_key": {
        "order": 0,
        "axis1": {
              "order": 0,
              "data": []...
```

Reply:
If not I can default to 0 in the front-end.

Reply:
This is a good point though, right now multi axis top events results would look like:
```
{
    "result_key": {
        "axis1": {
              "order": 0,
              "data": []...
```
so I'll move order up a level first to match below before merging :+1:
```
{
    "result_key": {
        "order": 0,
        "axis1": {
              "data": []...
```

Reply:
Right now we get

```json
{
  "rpm()": {"data": [....]},
  "apdex()": {"data": [....]},
}
```
Instead if we got

```
{
  "rpm()": {"order": 0, "data": [....]},
  "apdex()": {"order": 1, "data": [....]},
}
```

That would make top5 and multi-y-axis requests shaped the same.

Reply:
Misunderstood your original comment, can definitely shape both responses the same. But what happens in the case of multi axis + top events?